packages/google_maps_flutter/google_maps_flutter_android/pigeons/messages.dart

```diff
@@ -509,6 +504,73 @@ class PlatformZoomRange {
   final double? max;
 }
 
+class PlatformBitmap {
+  PlatformBitmap({required this.bitmap});
+
+  final Object bitmap;
+}
+
+class PlatformBitmapDefaultMarker {
```

Comment:
These should all have comments explaining what platform interface class they correspond to, with doc links.